Monika After Story/game/script-farewells.rpy

```diff
@@ -960,6 +979,25 @@ label bye_going_somewhere:
         m 2dsc "Just...please."
         m 2rkc "{i}Please{/i} understand what I'm going through."
         m 1dkc "..."
+        if mas_isMoniLove(higher=True):
```

Comment:
This is inside a `mas_isMoniUpset(lower=True):`, so it will never execute. 

It looks like everything else is just duplicated from the earlier segment, including some mistakes. Duplicated code generally isn't going to slide by.